pkg/kubectl/cmd/clusterinfo_dump.go

```diff
@@ -45,19 +47,20 @@ type ClusterInfoDumpOptions struct {
 	AllNamespaces bool
 	Namespaces    []string
 
-	Timeout          time.Duration
-	Clientset        internalclientset.Interface
-	Namespace        string
-	RESTClientGetter genericclioptions.RESTClientGetter
-	LogsForObject    polymorphichelpers.LogsForObjectFunc
+	Timeout             time.Duration
+	Clientset           corev1client.CoreV1Interface
+	ExtensionsClientset extensionsv1client.ExtensionsV1beta1Interface
```

Comment:
`DaemonSet`, `ReplicaSet` and `Deployment` are part of `apps/v1` since 1.10 at least. It's about time we switch commands to use the GA version.

Reply:
@charrywanganthony You'd need to switch out extensionsv1client for `appsv1client "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"`

Reply:
@juanvallejo WIP but `openshift/client-go` seems only have function for deployment `func (c *AppsV1Client) DeploymentConfigs(namespace string)` did i miss something?

Reply:
@charrywanganthony sorry about that, the actual import path for the appsv1 client is `"k8s.io/client-go/kubernetes/typed/apps/v1"`

Reply:
@juanvallejo  @soltysh modified.